types/koa-bunyan-logger/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for koa-bunyan-logger 2.0
+// Project: https://github.com/koajs/bunyan-logger
+// Definitions by: Steven McDowall <https://github.com/sjmcdowall>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { Middleware } from 'koa';
+
+import * as Logger from 'bunyan';
+
+export default function(logger?: Logger): Middleware;
```

Comment:
It seems like the library not using `export default`, so this .d.ts file should not either.

Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html).